daemon/lokinet.cpp

```diff
@@ -525,7 +525,7 @@ namespace
               "⠄⠄⠄⠄⠄⠿⠿⠟⠛⡹⠉⠛⠛⠿⠿⣿⣿⣿⣿⣿⡿⠂⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄",
               "⠠⠤⠤⠄⠄⣀⠄⠄⠄⠑⠠⣤⣀⣀⣀⡘⣿⠿⠙⠻⡍⢀⡈⠂⠄⠄⠄⠄⠄⠄⠄⠄⠄",
               "⠄⠄⠄⠄⠄⠄⠑⠠⣠⣴⣾⣿⣿⣿⣿⣿⣿⣇⠉⠄⠻⣿⣷⣄⡀⠄⠄⠄⠄⠄⠄⠄⠄",
-              "file a bug report now or be cursed with this "
+              "file a bug report now or be cursed with this ",
```

Comment:
I don't think the comma is supposed to be there -- that last line is meant to be one long line (i.e. we're wrapping the string in source code only, not in the output).

Reply:
fixed